deploy/overlays/stanford-staging/custom_bigtable_info.yaml

```diff
@@ -0,0 +1,4 @@
+project: datcom-stanford-staging
```

Comment:
The project here should be "datcom-stanford". The staging and prod instance share the same store project, which I think should be the prod project.

Reply:
Makes sense, thanks